README.md

```diff
@@ -101,17 +106,27 @@ This DSL relies on the Ruby [pact-mock_service][pact-mock-service] gem to provid
 
 1. Let's run that bad boy!
 
-   * start the pact mock server with `bundle exec pact-mock-service -p 1234 -l log/pact.logs --pact-dir tmp/pacts`
+   * start the pact mock server with `pact-mock-service -p 1234 -l log/pact.logs --pact-dir tmp/pacts`
    * run `karma start` (in another terminal window)
    * inspect the pact file that has been written to "hello_consumer-hello_provider.json"
 
-### Example
+#### Web Example
 
-Have a look at the [example](/example) folder. Ensure you have Google Chrome installed.
+Have a look at the [example](/example/web) folder. Ensure you have Google Chrome installed.
 
     $ cd example
     $ npm install
     $ script/test.sh
+    
+#### Nodejs Example
+
+1. Start pact-mock-service with `pact-mock-service -p 1234 -l tmp/pact.log --pact-dir tmp/pacts`
+
+1. Run nodejs command to setup pact consumer `node example/nodejs/setup.js`
+
+1. Run your tests here with whatever you want, like Protractor for e2e testing
+
+1. Run nodejs command to verify interactions and write pact files `node example/nodejs/teardown.js`
```

Comment:
Righto. If it works for you, that's great, but it's not the example I'd like to show people, because people will just unthinkingly copy stuff without critical thought.